api/SVGPathElement.json

```diff
@@ -14,10 +14,10 @@
             "version_added": "12"
           },
           "firefox": {
-            "version_added": true
+            "version_added": "1.5"
```

Comment:
Interesting, so `SVGPathElement` itself lives on, in a much smaller form:
https://github.com/mozilla/gecko-dev/blob/master/dom/webidl/SVGPathElement.webidl